### PR TITLE
Remove fs2 prefix

### PR DESCRIPTION
**Pull request by noelmarkham**
This was causing some headaches downstream in Mu, better to revisit this in its own release I think.

### CODE REVIEW

**Comment by pepegar** [comment]
This looks good.  Also, since this will end up in a macro, I would prefix all `fs2` packages with `_root_`, in order to not make it collide with user's packages called `fs2`.

What do you think?

**Comment by noelmarkham** [comment]
Sounds good - I think that should be done in a separate PR when addressing the `fs2.` prefix

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.10-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.0.9.2"
```
Comment (fedefernandez) [inline version.sbt]: ☝️ ⚠️

**Comment by pepegar** [comment]
@noelmarkham sounds good

File: version.sbt (hunk 1 above)
Comment (fedefernandez) [inline version.sbt]: hey @noelmarkham do you know if sbt-release handles the fourth number well. From [semantic versioning](https://semver.org/) additional labels are separated by `-`, which I think aren't supported by sbt-release.

**Comment by noelmarkham** [comment]
@pepegar I was seeing this as a patch on `0.0.9` - similarly `0.0.9.1` was purely a bump with no real changes as Sonatype had failed.

File: version.sbt (hunk 1 above)
Comment (noelmarkham) [reply version.sbt]: `0.0.9.1` was fine, but that was purely a nudge to make the build happen again. Considering this has actual code changes, I'll make it `0.0.10`.

**Pull request merged**